### PR TITLE
chore: introduce rfc process

### PR DESCRIPTION
**Pull request by scurker**
[Preview](/dequelabs/cauldron/blob/rfcs/rfcs.md)

### CODE REVIEW

File: rfcs.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+You should follow this process if you intend to make "substantial" changes to cauldron or its documentation. Some examples that would benefit from an RFC include:
+
+- A new component or feature
+- Any breaking changes
+
+Some changes that would not need an RFC:
+
+- Bug fixes
+- Refactoring an existing component (while maintaining the same API contract)
+- Rephasing or clarifying documentation
```
Comment (thuey) [inline rfcs.md:20]:
```suggestion
- Rephrasing or clarifying documentation
```

File: rfcs.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## How to submit an RFC
+
+1. Create a new issue
+1. Add the `rfc` label
+1. Be sure to include descriptive details, this includes:
+   - A descriptive title
+   - Implemention details
+   - Screenshots (if necessary for a new component)
```
Comment (thuey) [inline rfcs.md:29]: What do you think about creating an issue template for this?

**Review by thuey — DISMISSED** [review]

File: rfcs.md (hunk 2 above)
Comment (scurker) [reply rfcs.md:29]: That's a great idea!

**Review by thuey — DISMISSED** [review]

File: rfcs.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Cauldron RFCs
+
+## What is an RFC?
+
+A request for comments, or further referred to as "RFC" is a process intended to provide a consistent and controlled path for changes to be introduced to cauldron.
```
Comment (stephenmathieson) [inline rfcs.md:5]:
```suggestion
A request for comments, or further referred to as "RFC" is a process intended to provide a consistent and controlled path for changes to be introduced to Cauldron.
```

File: rfcs.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+Many changes, including bug fixes and documentation improvements can be implemented and reviewed via the normal GitHub pull request workflow. Some changes though are "substantial", and we ask that these be put through a bit of a design process and provide a consensus among the cauldron team.
```
Comment (stephenmathieson) [inline rfcs.md:7]:
```suggestion
Many changes, including bug fixes and documentation improvements can be implemented and reviewed via the normal GitHub pull request workflow. Some changes though are "substantial", and we ask that these be put through a bit of a design process and provide a consensus among the Cauldron team.
```

Should "Cauldron team" link to @dequelabs/cauldron-team?

File: rfcs.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+## When to follow this process
+
+You should follow this process if you intend to make "substantial" changes to cauldron or its documentation. Some examples that would benefit from an RFC include:
```
Comment (stephenmathieson) [inline rfcs.md:11]:
```suggestion
You should follow this process if you intend to make substantial changes to Cauldron or its documentation. Some examples that would benefit from an RFC include:
```

I don't think we need to wrap this in quotes a second time.

File: rfcs.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+1. Create a new issue
+1. Add the `rfc` label
+1. Be sure to include descriptive details, this includes:
+   - A descriptive title
+   - Implemention details
+   - Screenshots (if necessary for a new component)
+
+## The RFC Process
+
+The goal of our RFC process is to reach a consensus among the cauldron team on proposed changes, and to ensure that new features are congruent with the rest of cauldron. The cauldron team will have a recurring office hours meeting on the 1st Wednesday of each month, giving the team the opportunity to discuss any new RFCs that have been introduced and make comments.
```
Comment (stephenmathieson) [inline rfcs.md:33]:
```suggestion
The goal of our RFC process is to reach a consensus among the cauldron team on proposed changes, and to ensure that new features are congruent with the rest of Cauldron. The Cauldron team will have a recurring office hours meeting on the 1st Wednesday of each month, giving the team the opportunity to discuss any new RFCs that have been introduced and make comments.
```

**Review by stephenmathieson — APPROVED** [review]
Couple of nits, but LGTM

**Review by stephenmathieson — APPROVED** [review]

**Review by ryomtoob — APPROVED** [review]
Looks good! I like Trevor's idea of making it an issue template since that will make it more discoverable.

**Comment by scurker** [comment]
> Looks good! I like Trevor's idea of making an issue template for it since that will make it more discoverable.

I think we can follow up with that once we have more time. 🙃

**Pull request merged**